### ISSUE TITLE
[Metrics] Use mean correlation at the validation set metric

### ISSUE THREAD

**Issue by kapiblue** [JankowskiDaniel/Neural-Deep-Retina#94]
Use mean Pearson correlation coefficient for as the validation set metrics.

As correlation is a more relevant metric than MAE or MSE, we should see a better performance of early stopping, which currently tracks the validation loss.

**Comment by kapiblue** [comment]
Use pcorr also for early stopping and saving the best model.
